### ISSUE TITLE
Is it alive?

### ISSUE THREAD

**Issue by vldmkr** [cmotc/android_device_samsung_schS738c#2]
I am interested in this project. Can you provide me a built CM image (or other), pls? It seems all the links to download are dead on your page [cmotc.github.io](http://cmotc.github.io/). Thanks!

**Comment by cmotc** [comment]
No, it pretty much isn't. I no longer have a Centura and it no longer builds with a current Android source tree so there isn't much I can do to make it work again. I might be able to get back to work on it in a month or two but right now I just don't have enough time.

**Comment by cmotc** [comment]
I am pretty sure that the Recovery will still work just fine though. If you want to try and make it work on your device, that's the place to start, and I will help where I can.

**Comment by vldmkr** [comment]
Oh, I understood. Then I will start to work from building recoveryimage. If I will have problems that I cannot solve, I will ask you. Thanks!
